### ISSUE TITLE
Refresh token in Kyma Kubeconfig

### ISSUE THREAD

**Issue by piotrmsc** [kyma-project/kyma#5773]
**Description**

Developers in kyma are using kubeconfig downloaded in our kyma console. This kubeconfig allows them to perform an interaction with cluster without direct access to k8s apiserver. Currently kubeconfig contains only users ID Token which depending on configuration is valid for 8hrs.
Although 8hrs seems like a long period, from a security perspective it should be valid as short as possible. We cannot simply change ID Token period of validity to for example 15 mins because this would require users to fetch each 15 mins new kubeconfig. This is happening due to the fact that kyma kubeconfig does not have refresh tokens and configuration that allows to refresh the token automatically by kubectl.

As a Kyma developer it should be easy to fetch kyma Kubeconfig with a configuration that will automatically refresh the token without the need to fetch repeatedly new one.
There should be a new way for fetching kubeconfig, not from console which makes http cal to `iam-kubeconfig-service` because refreshing the token requires `clientSecret` to be present.
Instead, console should show sth like GCP popup with a command to execute via kyma cli to fetch kubeconfig.

**Reasons**
Currently kubeconfig is valid as long as id token is valid, there's no refresh option, despite the manual delete&download. This is poor developer experience.

**Technical context**

** Sample kubeconfig**

```

apiVersion: v1
kind: Config
clusters:
- name: some-cluster
  cluster:
    server: https://apiserver.kyma.local
    certificate-authority-data: CERT
contexts:
- name: oidc
  context:
    cluster: 
    user: oidcUser
users:
- name: oidcUser
  user:
    auth-provider:
      name: oidc
      config:
        client-id: {{ .ClientID }}
        client-secret: {{ .ClientSecret }}
        idp-issuer-url: {{ .Issuer }}
        refresh-token: {{ .RefreshToken }
```

**Comment by piotrmsc** [comment]
still valid

**Comment by piotrmsc** [comment]
Will be implemented as part of #5792

**Comment by muneendra-yeddala** [comment]
Hi,
Any plans to address this issue. 8 hours validity of token is sort of not allowing  us to automate the workload deployments and other configurations through kubectl.

Thanks
